noxfile.py

```diff
@@ -75,28 +57,19 @@ def lint(session):
     session.run("python", "-m", "build", "--sdist")
     session.run("twine", "check", "--strict", "dist/*")
 
-
 @nox.session()
 def format(session):
     """
-    Run isort to sort imports. Then run black
+    Run isort to sort imports. Then run black 
     to format code to uniform standard.
     """
-    session.install(BLACK_VERSION, ISORT_VERSION)
+    session.install("ruff")
     # Use the --fss option to sort imports using strict alphabetical order.
     # See https://pycqa.github.io/isort/docs/configuration/options.html#force-sort-within-sectionss
```

Comment:
can we update the docstring to mention ruff now and not isort/black. Also the comment for `"use the -fss..` no longer applies so it should be removed...